test/unittests/t_cache_ram.cc

```diff
@@ -23,7 +24,7 @@ namespace cache {
 class T_RamCacheManager : public ::testing::Test {
  public:
   T_RamCacheManager()
-    : ramcache_(4*alloc_size, 3*cache_size, &statistics_) {
+    : ramcache_(4*alloc_size, cache_size, &statistics_) {
```

Comment:
I only realize now, sorry: is the initialization done in the constructor for a reason?  The rest uses `Setup` and `TearDown`, which will be executed for each individual test.  But perhaps that's the same for the constructor...?